src/discof/repair/fd_repair_tile.c

```diff
@@ -424,6 +397,40 @@ fd_repair_send_requests( fd_repair_tile_ctx_t *   ctx,
 }
 
 
+static inline void
+handle_new_cluster_contact_info( fd_repair_tile_ctx_t * ctx,
+                                 uchar const *          buf,
+                                 ulong                  buf_sz ) {
+  fd_shred_dest_wire_t const * in_dests = (fd_shred_dest_wire_t const *)fd_type_pun_const( buf );
+
+  ulong dest_cnt = buf_sz;
+  if( FD_UNLIKELY( dest_cnt >= MAX_REPAIR_PEERS ) ) {
+    FD_LOG_WARNING(( "Cluster nodes had %lu destinations, which was more than the max of %lu", dest_cnt, MAX_REPAIR_PEERS ));
+    return;
+  }
+
+  /* Stop adding peers after we reach the peer max, but we may want to
+     consider an eviction policy. */
+  for( ulong i=0UL; i<dest_cnt; i++ ) {
+   if( FD_UNLIKELY( ctx->repair->peer_cnt >= FD_ACTIVE_KEY_MAX ) ) break;// FIXME: aiming to move all peer tracking out of lib into tile, leaving like this for now
+    fd_repair_peer_addr_t repair_peer = {
+      .addr = in_dests[i].ip4_addr,
+      .port = fd_ushort_bswap( in_dests[i].udp_port ),
+    };
+    int dup = fd_repair_add_active_peer( ctx->repair, &repair_peer, in_dests[i].pubkey );
+    if( !dup ) {
+      /*
+      Peer warm up. Peers send ping-pong messages on first repair request. Due to a design flaw in the iterator, we wait 40ms between each peer during warm up.
+      This is fix to workaround the iterator and decrease warm up time from ~40ms * num_peers to ~ time until first turbine. (about 50 seconds decrease in startup time)
+      */
```

Comment:
nit, but can you reformat to be a block comment capped out 72 columns formatted similarly to this?

https://github.com/firedancer-io/firedancer/blob/emwang/feat/store-pool-para/src/choreo/tower/fd_tower.c#L391-L394